py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -46,6 +46,24 @@ def aten_ops_batch_norm(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.cat.default)
+def aten_ops_cat(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.cat.cat(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        tensors=args[0],
+        dim=args_bounds_check(args, 2, 1),
```

Comment:
Switch to `dim=args_bounds_check(args, 1, 0)`, since the default should be 0 at the `args[1]` position, [as here](https://github.com/pytorch/pytorch/blob/a4dec8d306d96637aa4dc1ee9deba289b128c148/aten/src/ATen/native/native_functions.yaml#L1324).